packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -39,7 +42,7 @@ export class WorkspaceContextUtil {
 
     const bindedHandler = () => this.handleCliConfigChange();
     const cliConfigPath = join(
-      getRootWorkspacePath(),
+      workspaceUtils.getRootWorkspacePath(),
       SFDX_FOLDER,
```

Comment:
This can use `projectPaths.sfdxProjectConfig()`, ie:

 `const cliConfigPath = projectPaths.sfdxProjectConfig()`

Reply:
yup agreed.